### ISSUE TITLE
Add schedule

### ISSUE THREAD

**Issue by kracekumar** [pythonindia/inpycon2015#24]
@kirang89 Can you take up this and @vanishan Can you provide the required info ?

**Comment by kirang89** [comment]
Yes I will.

**Comment by harisibrahimkv** [comment]
@kirang89 Can you add a "Schedule" link at the top navigation bar? Between Speakers and Venue.

**Comment by kirang89** [comment]
Resolved with #35

**Comment by kracekumar** [comment]
Can we add end time in schedule ? As of now it says `10:30`, first timers don't know talks are for 45 minutes. We should say it as `10:45 - 11:30`.

**Comment by kracekumar** [comment]
On Oct 4, Tea Break is at 10:15 and not at 10:00

**Comment by kirang89** [comment]
Doesn't the time for the next batch of talks signify the end time for the previous one ?

**Comment by kirang89** [comment]
Tied up with work tonight. I'll get all the changes done tomorrow. Hope that's fine.

**Comment by kracekumar** [comment]
No. It is very implicit. Ideally we should say talks are 40 minutes. 5
minutes for setup and switch over the room.

On Fri, Sep 18, 2015 at 8:33 PM, Kiran Gangadharan <notifications@github.com

> wrote:
> 
> Tied up with work tonight. I'll get all the changes done tomorrow. Hope
> that's fine.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pythonindia/inpycon2015/issues/24#issuecomment-141477238
> .

## 

Regards
Kracekumar Ramaraju
http://kracekumar.com
+91 85530 29521

**Comment by kirang89** [comment]
Alright, I'll get it done.

**Comment by kirang89** [comment]
Resolved in #38
